source/agora/test/QuorumPreimage.d

```diff
@@ -49,7 +49,7 @@ unittest
     scope(exit) network.shutdown();
     // Disabled, as most of the time they just hide the more useful
     // quorum prints which is also a `scope (failure)` (see below)
-    // scope(failure) network.printLogs();
+    scope(failure) network.printLogs();
```

Comment:
Is the comment wrong nowadays ?

Reply:
Well, I am not sure what his intention was. but that comment is obviously no longer useful.